README.md

```diff
@@ -136,7 +140,9 @@ simple example could be:
 
 If you then run the following command in the directory `foo`:
 
-    $ swift build
+```sh
+$ swift build
+```
```

Comment:
Weird that none of the other shell commands include a `$` prompt. I'd get rid of it here to be consistent